framework/spree/api/operations/get-all-pages.ts

```diff
@@ -0,0 +1,82 @@
+import type {
+  OperationContext,
+  OperationOptions,
+} from '@commerce/api/operations'
+import type { GetAllPagesOperation, Page } from '@commerce/types/page'
+import { requireConfigValue } from '@framework/isomorphic-config'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers